sys/net/link_layer/iQueue-MAC/simple.c

```diff
@@ -0,0 +1,2082 @@
+/*
+ * Copyright (C) 2014  INRIA.
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup MAC
+ * @{
+ * @file    iqueue_MAC.c
+ * @brief   Implementation of the CoSenS MAC protocol for RIOT OS
+ * @details Implementation of the CoSenS MAC/radio duty cycling protocol;
+ *          designed by Shuguo Zhuo and Ye-Qiong Song, see:
+ *          Ad Hoc Networks 10 (2012) 680-695.
+ * @author  Kévin Roussel <Kevin.Roussel@inria.fr>
```

Comment:
Yes, I simply copied and pasted here. Actually, I'm working this with Kevin and Ye-qiong of INRIA, and I forgot to update the author information before uploading. Sorry for that. I will fix that next time.